### PR TITLE
Fix create comment and get activities methods

### PR DESCRIPTION
**Pull request by syndicut**

### CODE REVIEW

**Comment by gfleury** [comment]
Hey! Thanks for contributing! I left some minor comments.

**Comment by syndicut** [comment]
> Hey! Thanks for contributing! I left some minor comments.

Sorry, I don't see any comments, am I missing something?

**Comment by gfleury** [comment]
Sorry, forgot to publish the review. Actually is just one comment.

File: default_api.go [hunk 1]
```diff
@@ -724,14 +724,20 @@ func (a *DefaultApiService) CreateBranch(projectKey, repositorySlug string) (*AP
  @param optional (nil or map[string]interface{}) with one or more of:
 	 @param "since" (string) For a merge commit, a parent can be provided to specify which diff the comments should be on. For                  a commit range, a {@code sinceId} can be provided to specify where the comments should be                  anchored from.
  @return */
-func (a *DefaultApiService) CreateComment(projectKey, repositorySlug string, commitId string, localVarOptionals map[string]interface{}) (*APIResponse, error) {
+func (a *DefaultApiService) CreateComment(projectKey, repositorySlug string, commitId string, comment Comment, localVarOptionals map[string]interface{}) (*APIResponse, error) {
```
[the remark this answers is not in the document]
Comment (syndicut) [reply default_api.go:727]: Good point though I don's see the reason why anyone would use non-working method (it always posts empty body)

**Pull request merged**